### ISSUE TITLE
[0.9.5.1] time >= 0 yields strange results

### ISSUE THREAD

**Issue by francisdb** [influxdata/influxdb#5120]
Looks like `>=` is handled incorrectly

Using the docker image available at https://github.com/tutumcloud/influxdb/tree/master/0.9

Written data:

```
temperature,location=room1 value=20.0 0
temperature,location=room1 value=22.0 60000000000
temperature,location=room1 value=24.0 120000000000
temperature,location=room1 value=26.0 180000000000
```

query

```
SELECT MEAN("value") 
FROM "temperature" 
WHERE "location"='room1' AND time > 0ms AND time < 200000ms 
GROUP BY time(1m)
```

returns

```
time	mean
1970-01-01T00:00:00Z	
1970-01-01T00:01:00Z	22
1970-01-01T00:02:00Z	24
1970-01-01T00:03:00Z	26
```

but

```
SELECT MEAN("value") 
FROM "temperature" 
WHERE "location"='room1' AND time >= 0ms AND time < 200000ms 
GROUP BY time(1m)
```

returns

```
time	mean
1970-01-01T00:00:00Z	23
```

I would expect this case to return this instead

```
1970-01-01T00:00:00Z	20
1970-01-01T00:01:00Z	22
1970-01-01T00:02:00Z	24
1970-01-01T00:03:00Z	26
```

With `time >= 1ms` I again get the result with the first interval empty. So this only happens for the specific case of `time >= 0`

**Comment by peetonn** [comment]
I guess it relates to this one https://github.com/influxdb/influxdb/issues/5113

**Comment by beckettsean** [comment]
This is another special case of https://github.com/influxdb/influxdb/issues/2703.

**Comment by jsternberg** [comment]
We have completely replaced the query engine as of 0.11 so I'm going to close this since the issue likely does not exist anymore. Please try out 0.13 and comment here if this still happens. Thanks.

**Comment by beckettsean** [comment]
fixed in 0.13: 

```
> select * from tt
name: tt
--------
time	foo
0	42
1	43
2	44

> select * from tt where time > 0
name: tt
--------
time	foo
1	43
2	44

> select * from tt where time >= 0
name: tt
--------
time	foo
0	42
1	43
2	44


> select mean(foo) from tt
name: tt
--------
time	mean
0	43

> select mean(foo) from tt where time > 0
name: tt
--------
time	mean
1	43.5

> select mean(foo) from tt where time >= 0
name: tt
--------
time	mean
0	43


```

**Comment by francisdb** [comment]
great, thanks @beckettsean
